api/VnJIBrrM0KY3uQ9X/index.js

```diff
@@ -22,29 +24,59 @@ class _VnJIBrrM0KY3uQ9X extends Endpoint {
      * @returns {*}
      */
     handler(app, req, res) {
-        if (!req.query.p0) {
-            return res.status(400).send({
-                status: 'fail',
-                message: 'p0<transaction_payload_signed> is required'
-            });
+        let transaction;
+
+        if (req.method === 'GET') {
+            if (!req.query.p0) {
+                return res.status(400).send({
+                    status : 'fail',
+                    message: 'p0<transaction_payload_signed> is required'
+                });
+            }
+            else {
+                transaction = JSON.parse(req.query.p0);
+            }
         }
+        else {
+            transaction = req.body.p0;
```

Comment:
should verify if p0 exists
see GET:

if (!req.query.p0) {
            });	                return res.status(400).send({
                    status : 'fail',
                    message: 'p0<transaction_payload_signed> is required'
                });
            }